Translation.md

```diff
@@ -44,4 +44,75 @@ DISCOVER/_build/html/index.html
 ```
 and opening it in your browser.  
 
+
 > Note: Contributors working on multilingual support should ensure .po file updates are included in commits.
+
+
+## Translation Workflow with Transifex
+
+We use Transifex to manage translations for the DISCOVER Cookbook. This guide outlines when and how to interact with Transifex during development. You likely won’t need every step each time—focus on the ones relevant to your task.
+
+### Step 1: Installing the Transifex CLI
+
+Transifex provides a Go-based CLI for syncing translation files. Follow the (official installation instructions)[https://developers.transifex.com/docs/cli] for your operating system.
+
+
+Verify installation:
+```sh
+tx --version
+```
+Expected output:
+```
+TX CLient, version=1.6.x
+```
+
+### Step 2: Authenticate with Transifex
+
+To authenticate, create a ```.transifexrc``` file in your home directory. Refer to the [official authentication guide](https://developers.transifex.com/reference/api-authentication) for details.
+
+
+Get your token from [Transifex Account Settings](https://app.transifex.com/user/settings/api/).
+
+
+### Step 3: Updating ```.tx/config```
+
+The ```.tx/config``` file maps source files to Transifex resources. You’ll only need to modify this when:
+
+- Adding a new chapter or page to the Cookbook
+- Removing or renaming existing translation files
+
+Use the CLI to register new resources:
+```
+tx add \
+  --organization numfocus \
+  --project DISCOVER-Cookbook \
+  --resource <resource_name> \
+  --file-filter locales/<lang>/LC_MESSAGES/<filename>.po \
+  --type MARKDOWN \
+  locales/en/LC_MESSAGES/<filename>.po
+```
+
+Refer to the [Transifex CLI reference](https://developers.transifex.com/docs/cli) for more information.
+
+
+### Step 4: Pushing Source and Translation Files
+
+After updating content or adding new chapters:
+- Regenerate ```.po``` files using ```sphinx-gettext``` and ```sphinx-intl```.
+- Open a PR with the updated files.
+- Push to Transifex:
```

Comment:
at the call after @kamila-NF feedback we decided to do these steps only when a release is made, not every time a change is made